modules/layers/src/scatterplot-layer/scatterplot-layer-inputs.ts

```diff
@@ -0,0 +1,52 @@
+import {ShaderModule} from '@luma.gl/shadertools';
+const uniformBlock = `\
+uniform scatterplotUniforms {
+  uniform float radiusScale;
+  uniform float radiusMinPixels;
+  uniform float radiusMaxPixels;
+  uniform float lineWidthScale;
+  uniform float lineWidthMinPixels;
+  uniform float lineWidthMaxPixels;
+  uniform float stroked;
+  uniform bool filled;
+  uniform bool antialiasing;
+  uniform bool billboard;
+  uniform highp int radiusUnits;
+  uniform highp int lineWidthUnits;
+} scatterplot;
+`;
+
+export type ScatterplotSettings = {
+  radiusScale?: number;
+  radiusMinPixels?: number;
+  radiusMaxPixels?: number;
+  lineWidthScale?: number;
+  lineWidthMinPixels?: number;
+  lineWidthMaxPixels?: number;
+  stroked?: boolean;
+  filled?: boolean;
+  antialiasing?: boolean;
+  billboard?: boolean;
+  radiusUnits?: number;
+  lineWidthUnits?: number;
+};
+
+export default {
+  name: 'scatterplot',
+  vs: uniformBlock,
```

Comment:
Not something we need to sort now:
- I get the concern about typing twice and getting errors, but this extra module and file for just uniforms is just too much plumbing in my opinion that fragments code and adds as much inefficiency as it solves

There are other possible approaches we should consider,
- e.g. autogenerating the blocks from the shader module props.
- adding a `common`, or `uniforms` field to shader modules,
- etc.